assets/blocks/course-outline/data.js

```diff
@@ -115,3 +115,30 @@ export const extractStructure = ( blocks ) => {
 		} )
 		.filter( ( block ) => 'module' === block.type || !! block.title );
 };
+
+/**
+ * Get first block by name.
+ *
+ * @param {string} blockName Block name.
+ * @param {Array}  blocks    Blocks array.
+ *
+ * @return {Object|boolean} Block or false.
+ */
+export const getFirstBlockByName = ( blockName, blocks ) => {
```

Comment:
I think we could keep it like that just to prevent extra complexity to the store. I think we could implement that in the future if we need the outline reference more times, in different parts of the code. WDYT? But if you consider it very important, I can update it.

Reply:
I think that would make it a bit more clear and explicit, but not critical now.